src/test/java/de/tum/in/www1/artemis/quiz/QuizExerciseIntegrationTest.java

```diff
@@ -1216,9 +815,8 @@ void testPerformStartNow() throws Exception {
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     @EnumSource(value = QuizMode.class, names = { "SYNCHRONIZED" }, mode = EnumSource.Mode.EXCLUDE)
     void testPerformStartNow_invalidMode(QuizMode quizMode) throws Exception {
-        quizExercise = createQuizOnServer(ZonedDateTime.now().plusHours(5), null, quizMode);
+        QuizExercise quizExercise = database.createAndSaveQuiz(ZonedDateTime.now().plusHours(5), null, quizMode);
         quizExercise.setReleaseDate(ZonedDateTime.now().minusHours(5));
```

Comment:
This line now doesn't do anything. Previously it was saved in the DB, now it just hangs loose.

Reply:
The quizExercise is then sent in the PUT request, I'm not exactly sure what you mean

Reply:
Nevermind